changelog.md

```diff
@@ -1,11 +1,27 @@
+
+
 ## Changes from 0.11.0 to 0.12.0
 
-### Fixed Issues
+### Overview
 
-- #2294 - Make boolean command line flags use Scallop's 'toggle'
 
-### Details for Marathon UI
-The web UI ships with a significant number of changes.
+#### Enable extensions to Marathon via Plugins
+This version of Marathon ships with the ability to load and use external plugins.
+With this functionality in place, you can extend and adapt functionality in Marathon to your specific needs.
+We start this adventure with pluggable authentication and authorization hooks, but want extend this
+to various functionality inside Marathon. 
+
+To start with this feature, please read [Plugin Documentation](https://mesosphere.github.io/marathon/docs/plugin.html)
+or see our [Example Plugins](https://github.com/mesosphere/marathon-example-plugins).
+
+Please check it out and give us feedback!
+We are also interested in any recommendations for upcoming plugin hooks. 
+
+#### Pluggable Authentication and Authorization hooks
+The probably most wanted feature in Marathon is the ability to have Authentication and Authorization in Marathon.
+Since this topic has so versatile requirements for different organizations, it is a perfect match for our new plugin mechanism.
+This version now implements all the necessary hooks needed to secure all external interfaces to your specific needs.
```

Comment:
s/all/most/

We're still missing a few endpoints such as DELETE /v2/leader =).